generators/enablefpm/index.js

```diff
@@ -1,7 +1,50 @@
 const path = require("path");
 const Generator = require("yeoman-generator");
+const UI5Config = require("@sap-ux/ui5-config").UI5Config;
 const fpmWriter = require("@sap-ux/fe-fpm-writer");
 const utils = require("../utils");
+const { join } = require("path");
+
+/**
+ * UI5 tasks configurations required for TypeScript projects
+ */
+const ui5TsTasks = [
+    {
+        name: 'ui5-tooling-modules-task',
```

Comment:
do you add the tooling-modules because you need it or do you add it just to have it? I'm fine with having it, just want to understand it.

Reply:
Initially I thought it is required and when I realized that it isn't I thought it is still a good idea to keep it. I can also remove it, I am undecided

Reply:
removed them with https://github.com/ui5-community/generator-ui5-project/pull/58/commits/0fc2353416b540d0be8eed26993cc843f726cd8b